packages/quantic/force-app/main/translations/fr.translation-meta.xml

```diff
@@ -228,4 +228,20 @@
         <label>Agrandir {{0}}</label>
         <name>quantic_Expand</name>
     </customLabels>
+    <customLabels>
+      <label>Quel produit est lié à votre problème ?</label>
+      <name>quantic_CaseClassificationTitle</name>
+    </customLabels>
+    <customLabels>
+      <label>Sélectionner un produit</label>
+      <name>quantic_SelectProduct</name>
+    </customLabels>
+    <customLabels>
+      <label>Sélectionnez une autre option</label>
+      <name>quantic_SelectAnotherFeature</name>
+    </customLabels>
+    <customLabels>
+      <label>Sélectionner une option</label>
+      <name>quantic_SelectOption</name>
+    </customLabels>
```

Comment:
@acarpentier01 That's the French  text used in the case classification component. Let me know if there's any thing I can improve.